geo/benches/relate.rs

```diff
@@ -0,0 +1,33 @@
+#[macro_use]
+extern crate criterion;
+extern crate geo;
+
+use crate::geo::relate::Relate;
+use criterion::Criterion;
+use geo::{LineString, Polygon};
+
+fn criterion_benchmark(c: &mut Criterion) {
+    c.bench_function("relate medium.relate(medium)", |bencher| {
+        let points = include!("../src/algorithm/test_fixtures/norway_main.rs");
+        let sub_polygon = {
+            let points = points[0..50].to_vec();
+            let mut exterior = LineString::<f32>::from(points);
+            exterior.close();
+            Polygon::new(exterior, vec![])
+        };
+
+        let polygon = {
+            let points = points[30..90].to_vec();
+            let mut exterior = LineString::<f32>::from(points);
+            exterior.close();
+            Polygon::new(exterior, vec![])
+        };
+
+        bencher.iter(|| {
+            criterion::black_box(criterion::black_box(&polygon).relate(&sub_polygon));
```

Comment:
I recall in the depths of my brain that every parameter of the function your testing should be wrapped in a `black_box`, but I can't find a source for that now 🤷🏻 

Related to https://github.com/rust-lang/rust/issues/80480

Reply:
I've added another block_box around the argument. So now it's:

```
 criterion::black_box(criterion::black_box(&polygon).relate(criterion::black_box(&sub_polygon)));
```